app/client/components/dashboard/beneficiaryLiveFeed.jsx

```diff
@@ -106,7 +103,7 @@ class BeneficiaryLiveFeed extends React.Component {
                                     <UserSVG src="/static/media/exchange.svg"/>
                                     <UserGroup>
                                       <ClickableTopText onClick={() => this.navigateToAccount(transferAccountId)}>{sender_user_name} exchanged</ClickableTopText>
-                                      <BottomText><Highlight>{transferFromMoney}</Highlight> for
+                                      <BottomText><DarkHighlight>{transferFromMoney}</DarkHighlight> for
```

Comment:
minor style change on live feed entries where the currency amount had inconsistent highlight